Gemfile

```diff
@@ -7,66 +7,59 @@ gem 'rails', '4.2.7.1'
 gem 'activeadmin', '~> 1.0.0.pre4'
 gem 'api-pagination'
 gem 'attachinary'
+gem 'autoprefixer-rails'
+gem 'bootstrap-sass', '~> 3.3.5'
+gem 'central-support', git: 'https://github.com/Codeminer42/cm42-central-support.git', branch: 'master', require: 'central/support'
```

Comment:
if you add this at the beginning of the file, you won't need to pass the full url
```
git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
  "https://github.com/#{repo_name}.git"
end
```